### PR TITLE
CVE-2013-1996 and CVE-2011-1772

### PR DESCRIPTION
**Pull request by kel-clark**

### CODE REVIEW

File: cves/CVE-2011-1772.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Multiple vulnerabilities allowed attackers to execute XSS attacks through input vectors involving an action or method name.
```
Comment (MattThyng) [inline cves/CVE-2011-1772.yml]: More detail in this description would be nice. Remember, you are trying to explain this to someone who doesn't know software security. For example, you may want to explain XSS more.

File: cves/CVE-2011-1772.yml [hunk 2]
```diff
@@ -103,9 +103,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: The original code was unit tested, but was not properly tested with input vectors to check for vulnerabilities.
```
Comment (MattThyng) [inline cves/CVE-2011-1772.yml]: Did the fix involve adding more unit tests or improving existing unit tests?

File: cves/CVE-2011-1772.yml [hunk 3]
```diff
@@ -119,20 +119,20 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: This bug was discovered by Dr. Marian Ventuneac at Genworth Financial, likely through either vulnerability testing or a previous attack. After looking at the Jira entry and Apache releases, it is not clear exactly how this vulnerability was discovered.
```
Comment (MattThyng) [inline cves/CVE-2011-1772.yml]: You can add multiline comments as follows:
answer: |
      My multiline
      comment here

File: cves/CVE-2013-1966.yml [hunk 4]
```diff
@@ -45,7 +45,7 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: URL requests containing parameterswere not properly handled, allowing malicious code in the native OGNL language to be executed through injection via the url parameters.
```
Comment (MattThyng) [inline cves/CVE-2013-1966.yml]: Try to explain this in more layman's terms. For example, what does OGNL stand for?

File: cves/CVE-2011-1772.yml (hunk 3 above)
Comment (kel-clark) [reply cves/CVE-2011-1772.yml]: I wasn't aware of this, thanks! It definitely makes this a lot more readable.

File: cves/CVE-2011-1772.yml (hunk 1 above)
Comment (kel-clark) [reply cves/CVE-2011-1772.yml]: Fixed this in the most recent commit, it should make it easier to understand now.

File: cves/CVE-2011-1772.yml (hunk 2 above)
Comment (kel-clark) [reply cves/CVE-2011-1772.yml]: No tests were changed regarding this fix, and that's been updated in the most recent commit.

File: cves/CVE-2013-1966.yml (hunk 4 above)
Comment (kel-clark) [reply cves/CVE-2013-1966.yml]: Updated with a definition of OGNL and a more simple explanation of XSS injection.

**Pull request merged**